storage/google/cloud/storage/client.py

```diff
@@ -316,21 +316,47 @@ def download_blob_to_file(self, blob_or_uri, file_obj, start=None, end=None):
 
         Args:
             blob_or_uri (Union[ \
-                :class:`~google.cloud.storage.blob.Blob`, \
-                 str, \
+            :class:`~google.cloud.storage.blob.Blob`, \
+             str, \
             ]):
                 The blob resource to pass or URI to download.
             file_obj (file):
                 A file handle to which to write the blob's data.
             start (int):
-                Optional, the first byte in a range to be downloaded.
+                Optional. The first byte in a range to be downloaded.
             end (int):
-                Optional, The last byte in a range to be downloaded.
+                Optional. The last byte in a range to be downloaded.
+
+        Examples:
+            Download a blob using using a blob resource.
+
+            >>> from google.cloud import storage
+            >>> client = storage.Client()
+
+            >>> bucket = client.get_bucket('my-bucket-name')
+            >>> blob = storage.Blob('path/to/blob', bucket)
+
+            >>> with open('file-to-download-to') as file_obj:
+            >>>     client.download_blob_to_file(blob, file)  # API request.
+
+
+            Download a blob using a URI.
+
+            >>> from google.cloud import storage
+            >>> client = storage.Client()
+
+            >>> with open('file-to-download-to') as file_obj:
+            >>>     client.download_blob_to_file(
+            >>>         'gs::/bucket_name/path/to/blob', file)
```

Comment:
Typo. Should be two slashes, one colon. `gs://`

Reply:
Fixed typo!